### PR TITLE
Add line to update data

### PR DESCRIPTION
**Pull request by nikosbosse**

### CODE REVIEW

File: jobs/germany.covid.forecasts/crowd-data.sh [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # check repository is up to date and update dependencies
 . schedule/jobs/germany.covid.forecasts/pull.sh
 
+# update data
+docker exec -w /home/rstudio/covid.german.forecasts forecast RscriptRscript data-raw/update.R
+
 # Redeploy forecast app (to update data)
 docker exec -w /home/rstudio/covid.german.forecasts forecast Rscript crowd-forecast/redeploy.R
```
Comment (seabbs) [inline jobs/germany.covid.forecasts/crowd-data.sh:10]: Shouldn't redeploy call this itself?

File: jobs/germany.covid.forecasts/crowd-data.sh [hunk 2]
```diff
@@ -3,6 +3,9 @@
 # check repository is up to date and update dependencies
 . schedule/jobs/germany.covid.forecasts/pull.sh
 
+# update data
+docker exec -w /home/rstudio/covid.german.forecasts forecast RscriptRscript data-raw/update.R
```
Comment (seabbs) [inline jobs/germany.covid.forecasts/crowd-data.sh:7]: Rscript duplicate

**Review by seabbs — CHANGES_REQUESTED** [review]
Dup on Rscript and I think data update should perhaps be called in redeploy (as it presumably always needs to happen for a redeployment)

File: jobs/germany.covid.forecasts/crowd-data.sh (hunk 1 above)
Comment (nikosbosse) [reply jobs/germany.covid.forecasts/crowd-data.sh:10]: there is no data update on Tuesdays. But also happy to move that into redeploy.

Comment (seabbs) [reply jobs/germany.covid.forecasts/crowd-data.sh:10]: hmm good point. Makes sense to have here then or have a second update bash script in crowd-forecast (I am keen on having all the code in one place so it makes sense for others as much as possible).

**Review by seabbs — APPROVED** [review]

**Comment by seabbs** [comment]
I still think it would be better to have as many of these as scripts in the `covid.germany.forecasts` repo.

**Pull request merged**